auth-web/src/views/auth/SigninView.vue

```diff
@@ -39,13 +39,15 @@ export default class Signin extends Mixins(NextPageMixin) {
   @Prop({ default: '' }) redirectUrlLoginFail: string
 
   private async authenticationComplete () {
+    await this.loadUserInfo()
```

Comment:
Don't need await here - `loadUserInfo` is not an asynchronous call.

Reply:
actually, we need that, that was the issue that the currentuser value were not setting before the call. either await or settimeout solved that.